modules/tinymce/src/plugins/fullscreen/main/ts/Plugin.ts

```diff
@@ -19,6 +20,7 @@ export default (): void => {
     Options.register(editor);
     Commands.register(editor, fullscreenState);
     Buttons.register(editor, fullscreenState);
+    Events.setup(editor, fullscreenState);
```

Comment:
Why do we need this as well as the core feature? I expected one or the other, not both?